server.js

```diff
@@ -1,27 +1,80 @@
 import express from "express";
 import cors from "cors";
+import listEndpoints from "express-list-endpoints";
+
+import booksData from "./data/books.json";
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// import avocadoSalesData from "./data/avocado-sales.json";
-// import booksData from "./data/books.json";
-// import goldenGlobesData from "./data/golden-globes.json";
-// import netflixData from "./data/netflix-titles.json";
-// import topMusicData from "./data/top-music.json";
-
-// Defines the port the app will run on. Defaults to 8080, but can be overridden
-// when starting the server. Example command to overwrite PORT env variable value:
-// PORT=9000 npm start
 const port = process.env.PORT || 8080;
 const app = express();
 
-// Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
-// Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(listEndpoints(app));
+});
+
+//1. endpoint that returns the whole array of book ratings.
+app.get("/bookratings", (req, res) => {
+  res.status(200).json({
+    data: booksData,
+    success: true,
+  });
+});
+
+// 2. endpoint that returns one rated book by its title.
+app.get("/bookratings/title/:title", (req, res) => {
```

Comment:
I wonder if you have to have /bookratings/ for this specific endpoint? Genuine questions that I am curious about! :)